### ISSUE TITLE
check.assert.maybe.array.of.unemptyString throws when called with a null or undefined value.

### ISSUE THREAD

**Issue by ceineke** [philbooth/check-types.js#59]
Version: 5.0.0

Observed behavior:
```javascript
> check.assert.maybe.array.of.nonEmptyString(null);
Error: Invalid string
```

Expected behavior:
```javascript
> check.assert.maybe.array.of.nonEmptyString(null);
undefined
```

**Comment by philbooth** [comment]
Thanks for raising this, I'll dig into it over the weekend.

**Comment by philbooth** [comment]
Fixed and published as `5.1.0`.

**Comment by ceineke** [comment]
Thanks!
